offchain/danaswap-api/src/Ctl/Utils/HsmWallet.purs

```diff
@@ -50,6 +50,7 @@ makeHsmWallet = do
     { address
     , selectCollateral
     , signTx: hsmSignTx plutusPubKey
+    , signData: \_ _ -> unsafeCoerce "I didn't implement sign data for yubihsm, it probably isn't that hard" -- TODO implement this?
```

Comment:
Can you document here the implications of not having this implemented?

Reply:
There are plutus functions for checking signatures. I assume ctl added support for signing arbitrary data that a contract's logic might then use, we won't be able to take advantage of that without implementing this.